### DISCUSSION TITLE
add new cipher ssl

### DISCUSSION THREAD

**Discussion by blaisemichel** [ipxe/ipxe#1102] [General]
Hello 

when is it planned to add new ciphers to the ipxe product?  because actually  all cipher are weak

**Comment by mcb30** [comment]
iPXE supports TLS_DHE_RSA_WITH_AES_128_GCM_SHA256 and TLS_DHE_RSA_WITH_AES_256_GCM_SHA384, which both provide AEAD and forward secrecy.

You are welcome to offer funding for the development of further cipher suites.

**Reply by sgrigorov** [reply]
What funding would be sufficient for implementing ECDHE?

**Reply by mcb30** [reply]
ECDHE is now supported
